### PR TITLE
Display default page

### LINKED ISSUE #7: Display Default Page
Update reminder component to have a conditional that will only display notes if notes exist.

There should be a default welcome page welcoming users to the app and/or directions to create a new note. Feel free to be creative here.

### PR DESCRIPTION
**Pull request by dshendrickson**
## Purpose

Update reminder component to have a conditional that will only display notes if notes exist.

There should be a default welcome page welcoming users to the app and/or directions to create a new note. Feel free to be creative here.

## Approach

Added an 'index' route and added a template index.hbs and associated content.  Had to rename the accptenance test file back to 'reminder-test.js'.  The renamed file was not being found by the test server.

## Issues

closes #7

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -7,10 +7,11 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders', {path: '/reminders'}, function() {
+  this.route('reminders', function() {
     this.route('reminder', {path: '/:reminder_id'});
-    this.route('new', {path: '/new'});
+    this.route('new');
   });
+  this.route('index', {path: '/'});
```
Comment (nfosterky) [inline app/router.js]: You do not need to define a route for index
https://guides.emberjs.com/v2.10.0/routing/defining-your-routes/#toc_index-routes

File: app/routes/index.js [hunk 2]
```diff
@@ -1,7 +1,4 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-  beforeModel() {
-    this.transitionTo('reminders');
```
Comment (nfosterky) [inline app/routes/index.js]: Index should still reroute to reminders

File: app/templates/index.hbs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+<article class="welcome-text">
+  Welcome to remEmber! In case you've already forgotten, you're here to track what you'll forget to do. Let's get started!
+</article>
+
+<br />
+
+{{#link-to "reminders"}}
+  <button class="dashboard-redirect">remEmber Dashboard</button>
+{{/link-to}}
+
```
Comment (nfosterky) [inline app/templates/index.hbs]: Great template, it should go in the reminders index route not the root application index
https://guides.emberjs.com/v2.10.0/routing/defining-your-routes/#toc_index-routes

File: tests/acceptance/reminder-test.js [hunk 4]
```diff
@@ -13,15 +13,14 @@ test('viewing the homepage', function(assert) {
...
-    assert.equal(Ember.$('.spec-reminder-item').length, 5);
+    assert.equal(currentURL(), '/');
   });
 });
 
 test('clicking on an individual item', function(assert) {
   server.createList('reminder', 5);
 
-  visit('/');
+  visit('/reminders');
```
Comment (nfosterky) [inline tests/acceptance/reminder-test.js]: After fixing your routing you should be able to change this back to `visit('/');`

File: tests/acceptance/reminder-test.js [hunk 5]
```diff
@@ -32,7 +31,7 @@ test('clicking on an individual item', function(assert) {
 
 test('clicking on add new reminder', function(assert) {
 
-  visit('/');
+  visit('/reminders');
```
Comment (nfosterky) [inline tests/acceptance/reminder-test.js]: See above...

File: app/routes/index.js (hunk 2 above)
Comment (andrewLcrist) [inline app/routes/index.js]: But with this, it doesn't hit our welcome page first. Don't we want it to hit our welcome then the user can be redirected to the reminders list on click?

File: app/templates/index.hbs (hunk 3 above)
Comment (andrewLcrist) [inline app/templates/index.hbs]: We're not sure what you mean here...

Comment (nfosterky) [inline app/templates/index.hbs]: Each route has an index route by default. This template should go under your reminders route. (e.g. reminders/index.hbs)

Comment (andrewLcrist) [inline app/templates/index.hbs]: image

So we should have an index.hbs under `templates/remindersreminder`?

**Pull request merged**